src/style/ButtonComponents.css

```diff
@@ -0,0 +1,43 @@
+.docButton {
+    background-color: #2c4f85;
+    text-transform: none;
+    font-family: 'Open Sans';
+    font-size: 0.875rem;
+    flex-grow: 0;
+    font-weight: 700;
+    color: #ffffff;
+    opacity: 0.75;
+    text-shadow: 1px 1px 2px #403f4c;
+  }
+.docButton:hover {
+    color: rgba(188,194,206, 1);
+    background-color: transparent;
+}
+
+.configButton {
+    background-color: #2c4f85;
+    text-transform: none;
+    font-family: 'Open Sans';
+    font-size: 0.875rem;
+    flex-grow: 0;
+    font-weight: 700;
+    color: #ffffff;
+    opacity: 0.75;
+    text-shadow: 1px 1px 2px #403f4c;
+  }
+.configButton:hover {
+    color: rgba(188,194,206, 1);
+    background-color: #2c4f85;
+}
```

Comment:
I just talked to @SamCitron about this, but it looks like we're not actually using the `configButton` classes. If we don't need them any more, we should remove them. Also, we should rename this file to be more descriptive since it includes `versionText`. Maybe something like `FshSchoolStyles.css`?